### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Rohitprasad83**
-  changed default user to Rohit
- removed useAuth
- added title for each page
- filter from homepage
- icons change on like video and watch later

### CODE REVIEW

**Review by anupkgurung — APPROVED** [review]
looks good.
Clean code.

**Review by Aditi0505 — APPROVED** [review]
Clean code
Looks good to merge

File: src/pages/playlist/Playlist.jsx [hunk 1]
```diff
@@ -30,10 +28,11 @@ export function Playlist() {
             </h2>
           )}
           {!loading &&
+            playlists &&
             playlists.map(playlist => (
               <PlaylistCard playlist={playlist} key={playlist._id} />
             ))}
-          {!loading && playlists.length < 1 && (
+          {!loading && playlists && playlists.length < 1 && (
             <div className="add-videos">
```
Comment (HeyNitin) [inline src/pages/playlist/Playlist.jsx:36]: Any reason behind using the different formats of class names? some places its joined with  '__' , here it's '-' ?

File: src/pages/playlist/SinglePlaylist.jsx [hunk 2]
```diff
@@ -1,22 +1,23 @@
-import React, { useState, useEffect } from 'react'
+import React, { useEffect } from 'react'
 import { useParams, useNavigate } from 'react-router-dom'
 import {
   deletePlaylist,
   getAllPlaylistVideos,
 } from 'features/playlist/playlistSlice'
```
Comment (HeyNitin) [inline src/pages/playlist/SinglePlaylist.jsx:6]: It's good that you're using absolute imports

File: src/pages/singlevideo/SingleVideo.jsx [hunk 3]
```diff
@@ -1,37 +1,45 @@
 import { useState, useEffect } from 'react'
 import { Navbar, Footer, VideoCard } from 'components'
-import { useParams, useNavigate, Link } from 'react-router-dom'
+import { useParams } from 'react-router-dom'
 import { videos } from 'backend/db/videos'
-// import { addPlaylist, deletePlaylistVideo, addVideoToPlaylist } from 'services'
```
Comment (HeyNitin) [inline src/pages/singlevideo/SingleVideo.jsx:5]: you can removed this code if not necessary

**Review by HeyNitin — APPROVED** [review]
Good use of absolute imports.
You can try to keep consistency in class name,
There's some commented code, which can be removed.

Also, try adding screenshots in description or generate a preview website for better code reviews

File: src/pages/playlist/Playlist.jsx (hunk 1 above)
Comment (Rohitprasad83) [reply src/pages/playlist/Playlist.jsx:36]: the double underscore( __ ) is from my component library, which I will change to single dash(-).

File: src/pages/singlevideo/SingleVideo.jsx (hunk 3 above)
Comment (Rohitprasad83) [reply src/pages/singlevideo/SingleVideo.jsx:5]: sure

**Pull request merged**